demisto_sdk/yaml_tools/unifier.py

```diff
@@ -140,16 +139,13 @@ def get_code_file(self, script_type):
         :rtype: str
         """
 
-        ignore_regex = (r'CommonServerPython\.py|CommonServerUserPython\.py|demistomock\.py|test_.*\.py|_test\.py'
-                        r'|conftest\.py')
-        if not self.package_path.endswith('/'):
-            self.package_path += '/'
-        if self.package_path.endswith('Scripts/CommonServerPython/'):
-            return self.package_path + 'CommonServerPython.py'
-
-        script_path = list(filter(lambda x: not re.search(ignore_regex, x),
-                                  glob.glob(self.package_path + '*' + script_type)))[0]
-        return script_path
+        # We assume here the code file has the same name as the integration/script name, plus the addition of the type
+        package_name = os.path.basename(os.path.dirname(self.package_path))
+        code_file_path = self.package_path + package_name + script_type
+        if not os.path.isfile(code_file_path):
+            raise Exception('Code file does not exists or has different name than {}'
+                            .format(package_name + script_type))
+        return code_file_path
```

Comment:
What is this change?

Reply:
In order to make our automation better we should assume assumptions, talked to @yaakovi and we decided to refactor it that way.